### PR TITLE
get operator list from OperandRegistry

### PR DESCRIPTION
**Pull request by ZhuoxiLi**

### CODE REVIEW

File: controllers/goroutines/operator_status.go [hunk 1]
```diff
@@ -40,23 +40,16 @@ func UpdateCsCrStatus(bs *bootstrap.Bootstrap) {
...
-			"ibm-ingress-nginx-operator",
-			"ibm-licensing-operator",
-			"ibm-management-ingress-operator",
-			"ibm-mongodb-operator",
-			"ibm-monitoring-grafana-operator",
 			"ibm-namespace-scope-operator",
-			"ibm-platform-api-operator",
-			"ibm-zen-operator",
-			"operand-deployment-lifecycle-manager-app"}
+			"operand-deployment-lifecycle-manager-app"}...)
```
Comment (Daniel-Fan) [inline controllers/goroutines/operator_status.go:52]: Would we include the `ibm-cloud-operator` as well?

Comment (ZhuoxiLi) [reply controllers/goroutines/operator_status.go:52]: actually, I am wondering will we add crossplane operator and cloud operator to operandregistry in the futurn? maybe @horis233

Comment (horis233) [reply controllers/goroutines/operator_status.go:52]: No, they won't be added into operandregistry. We can add both of them to this list.

Comment (ZhuoxiLi) [reply controllers/goroutines/operator_status.go:53]: thank you all! added to the list

**Review by horis233 — APPROVED** [review]
/lgtm

**Pull request merged**